venom.go

```diff
@@ -2,8 +2,14 @@ package venom
 
 import (
 	"fmt"
+	"os"
 )
 
+// Version of Blinkstick
+// One Line for this, used by release.sh script
+// Keep "const Version on one line"
+const Version = "0.9.0"
```

Comment:
but why set 0.9 ? i would prefer "snapshot", if it's replaced on build time with tag name.

the very first public stable (1.0.0) will be released when we will considered it as stable - from a functionnal and design point of view (with plugin support for instance).